src/idiomatic/foundations-api-design/predictable-api/naming-conventions/12-to.md

```diff
@@ -0,0 +1,29 @@
+---
+minutes: 2
+---
+
+# To
+
+Prefix to a function that takes a borrowed value and creates an owned value
+
+```rust
+String::to_owned // &str -> String (&str is not consumed)
+```
+
+<details>
+- Method for a function from a borrowed value to owned value.
+
+Not an escape hatch to make something borrowed into something owned, instead
+creates a new owned value.
+
+May transform the data in some way! May return an entirely different type. This
+is what makes it different from clone.
+
+Example: to_uppercase creates a version of a string with all uppercase letters.
+
+- Tends to not consume source value.
```

Comment:
```suggestion
- If you want to define a method that consumes the source value, use the "into" naming pattern.
```

Why tends? It should not consume the source value. If it does, it should be an "into".